addons/web_editor/static/src/js/backend/field_html.js

```diff
@@ -507,6 +507,11 @@ var FieldHtml = basic_fields.DebouncedField.extend(TranslatableFieldMixin, {
             top: '+5px',
         });
         this.$el.append($button);
+        var showLinkDialog = $.summernote.eventHandler.modules.linkDialog.showLinkDialog;
+        $.summernote.eventHandler.modules.linkDialog.showLinkDialog = function ($editable, $dialog, linkInfo) {
+            linkInfo.forceNewWindow = true;
```

Comment:
I see the link dialog has this as constructor: `init: function (parent, options, editable, linkInfo) {` should not this be part of `options` instead of `linkInfo` ?

Reply:
Yes, I'm sure it would be clearer : We could notice directly that this information comes not directly from the link.

But... I wonder how I could design it.

Since the linkDialog is instanciate in SummernoteManager : https://github.com/odoo/odoo/blob/9fcaeb96ef8216a3590d56c2866892e3bb882040/addons%2Fweb_editor%2Fstatic%2Fsrc%2Fjs%2Feditor%2Frte.summernote.js#L1213-L1236

1) I would have to extend and use SummernoteManager which assets are loaded in a deferred call (if I understood well) and override _onLinkDialogDemand ? => Doesn't seem to be a great idea
2) Rewrite : https://github.com/odoo/odoo/blob/9fcaeb96ef8216a3590d56c2866892e3bb882040/addons%2Fweb_editor%2Fstatic%2Fsrc%2Fjs%2Feditor%2Frte.summernote.js#L441-L465 to use an 'option' field in the link_dialog_demand trigger on Line 452 (this field is read in the _onLinkDialogDemand but not used as I understood) ? Then, find a way to tune it nicely.

And, ofc, I find my way to redefine : $.summernote.eventHandler.modules.linkDialog.showLinkDialog absolutely :-1: So I would be happy to find a more sweet way to implement it.

Reply:
(Nevermind, I give an other try :) )